screens/Home.js

```diff
@@ -27,16 +27,14 @@ import Spinner from '../components/Spinner';
 import EchoLogo from '../components/EchoLogo';
 import HomeCardPlaceholder from '../components/HomeCardPlaceholder';
 
-const trackingOptions = {
-  app: 'mobile',
-  mainTray: 'Home',
-};
-
 const storePostsData = async data => {
   await AsyncStorage.setItem('@posts', JSON.stringify(data)).catch(err =>
     console.error(err)
   );
 };
+const getStoredPosts = () => {
+  return AsyncStorage.getItem('@posts').catch(err => console.error(err));
```

Comment:
Not tracking this error?

Reply:
nawww...it should work 👍  but we don't want it to break anything if it doesn't return any stored data just in case